packages/interval/index.ts

```diff
@@ -1,3 +1,38 @@
+type IntervalName = string;
+type IntervalQuality = string;
```

Comment:
I think this case can be narrowed:
```suggestion
type IntervalQuality = "dddd" | "ddd" | "dd" | "d" | "m" | "M" | "P" | "A" | "AA" | "AAA" | "AAAA";
```